core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -24,16 +24,25 @@ package kafka.api
  *
  * Note that the ID we initialize for each version is important.
  * We consider a version newer than another, if it has a higher ID (to avoid depending on lexicographic order)
+ * 
+ * If there is a draft protocol version between protocols of two official releases. The suffix "-IV#" will be added to
+ * help users who are running on trunk upgrade. For example:
+ * 1. Kafka 0.9.0 is released
+ * 2. After that some protocol change are made and will be released in 0.10.0. The version will be named 0.10.0-IV0
+ *    (DV stands for internal version)
```

Comment:
typo DV

Also, could we leave some comments so that people know what to do when changing the protocol again before 0.10.0 is released?